<?xml version="1.0"?>
<launch>
    <node pkg="path_planner" type="path_planner_node" name="path_planner_node" output="screen">
        <!-- 分别使用一号点和三号点进行标定 -->
        <param name="uavl" type="double" value="0.3"/>
        <param name="uavw" type="double" value="0.2"/>
        <param name="uavh" type="double" value="0.5"/>
        <param name="octo_resolution" type="double" value="0.1"/>
        <param name="bound_xy" type="double" value="8000.0"/>
        <param name="bound_lowz" type="double" value="0.0"/>
        <param name="bound_highz" type="double" value="1"/>
        <param name="step_range" type="double" value="0.2"/>
    </node>
</launch>
